experiments/axon_geometry/notebooks/linear_regression_length_segment.ipynb

```diff
@@ -0,0 +1,286 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Plot linear regression fits\n",
+    "\n",
+    "N.B. Data should be placed in `/brainlit/data/poster/reproduction`"
```

Comment:
Let's remove this cell that says "N.B. Data should be place in ..."

Reply:
Done